modules/op-setting-a-cron-job-for-refreshing-catalog-in-tekton-hub.adoc

```diff
@@ -0,0 +1,71 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="setting-a-cron-job-for-refreshing-catalog-in-tekton-hub_{context}"]
+= Optional: Setting a cron job for refreshing catalog in {tekton-hub}
+
+[role="_abstract"]
+Cluster administrators can optionally set up a cron job to refresh the database after a fixed interval, so that changes in the catalog reflects in the UI.
```

Comment:
```suggestion
Cluster administrators can optionally set up a cron job to refresh the database after a fixed interval so that changes in the catalog reflects in the UI.
```